packages/calcite-components/src/components/tip/tip.scss

```diff
@@ -1,23 +1,38 @@
+@import "../../assets/styles/header";
+
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-border-color: The border color of the component.
+* @prop --calcite-tip-background-color: The background color of the component.
+
+*/
+
 :host {
-  @apply border-color-2
-    bg-foreground-1
-    text-color-2
+  @apply text-color-2
```

Comment:
replace this with `color: var(--calcite-tip-text-color);` and add `--calcite-tip-text-color: var(--calcite-color-text-2)`